src/Controllers/OAuthController.php

```diff
@@ -32,7 +32,8 @@ public function redirectOAuth(string $oauthName): RedirectResponse
             return redirect()->to(config('Auth')->loginRedirect());
         }
 
-        if (config('ShieldOAuthConfig')->oauthConfigs[$oauthName]['allow_login'] === false) {
+        /** @phpstan-ignore-next-line */
```

Comment:
Instead of using /** @phpstan-ignore-next-line */ to suppress the error, you can fix it by making the following change:

```yaml
scanDirectories:
    - vendor/codeigniter4/settings/src/Helpers
```
This will allow PHPStan to properly scan the necessary directories and resolve the issue.

Reply:
Thank you for your guidance. I have sent an update. please review it.